### PR TITLE
feat: `bring messagefanout`

### PR DESCRIPTION
**Pull request by marciocadev**
This library connects a topic (SNS) directly to one or more queues (SQS), functioning as a 'message-fanout'.

To add a consumer (SQS) to this component, the preflight method is used:
`addConsumer(handler: inflight(str): void, props: MessageFanoutProps): void;`

To send a message, the inflight method is used:
`inflight publish(message: str): void;`

### This implementation supports the `tf-aws` and `sim` targets

### CODE REVIEW

File: message-fanout/platform/sim.w [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+bring "../commons/api.w" as api;
+
+pub class MessageFanout_sim impl api.IMessageFanout {
+  topic: cloud.Topic;
+
+  new() {
+    this.topic = new cloud.Topic();
+  }
+
+  pub addConsumer(handler: inflight(str): void, props: api.MessageFanoutProps): void {}
```
Comment (hasanaburayyan) [inline message-fanout/platform/sim.w:11]: So this does not do anything in the sim currently?

File: message-fanout/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+npm i @winglibs/message-fanout
+`
+
+## Usage
+
+`js
+bring message-fanout;
+
+let adder = new message-fanout.Adder();
+`
```
Comment (hasanaburayyan) [inline message-fanout/README.md:19]: can you add some examples of this adding consumers

File: message-fanout/platform/sim.w (hunk 1 above)
Comment (marciocadev) [reply message-fanout/platform/sim.w]: Sorry Hasan, I had forgotten to implement it, now it's correct

File: message-fanout/README.md (hunk 2 above)
Comment (marciocadev) [reply message-fanout/README.md]: Now it has a better example, Hasan.

**Review by hasanaburayyan — APPROVED** [review]
@marciocadev Thanks for fixing up the sim impl 🔥 LGTM

**Pull request merged**